WordPress/src/main/java/org/wordpress/android/ui/accounts/SignInFragment.java

```diff
@@ -904,6 +904,23 @@ protected void signIn() {
         }
     }
 
+    private void onWPComEmailCheckError(boolean forceWordPressComDisplay) {
+        if (!isAdded()) {
+            return;
+        }
+
+        if (forceWordPressComDisplay) {
```

Comment:
Is this `IF` condition correct? The guard says force wpcom display, but later we show the self hosted sign in form.